docs/modeling/units.rst

```diff
@@ -147,6 +147,7 @@ generating synthetic data:
 
     import numpy as np
     from astropy import units as u
+    import matplotlib.pyplot as plt
```

Comment:
@astrofrog @drdavella - I wonder why this hasn't been an issue for the directive?

Reply:
Because numpy and matplotlib imports are included by default (https://github.com/matplotlib/matplotlib/blob/v2.1.0/lib/matplotlib/sphinxext/plot_directive.py#L517-L518) if the `pre_code` option isn't specified.

However since these examples are meant to be standalone it's good to have them there explicitly.

Reply:
thanks for the confirmation. Would be nice if their doc would mention that, too.

Reply:
I'll make a PR upstream.

Reply:
thanks